modules/twinklewarn.js

```diff
@@ -1261,13 +1271,13 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 			break;
 		case 'kitchensink':
 			['level1', 'level2', 'level3', 'level4', 'level4im'].forEach(function(lvl) {
-				$.each(Twinkle.warn.messages.levels, function(_, levelGroup) {
-					createEntries(levelGroup, sub_group, true, lvl);
+				$.each(Twinkle.warn.messages.levels, function(levelGroupLabel, levelGroup) {
+					createGroup(levelGroup, 'Level' + lvl.slice(5) + ': ' + levelGroupLabel, true, lvl);
 				});
 			});
-			createEntries(Twinkle.warn.messages.singlenotice, sub_group, true);
-			createEntries(Twinkle.warn.messages.singlewarn, sub_group, true);
-			createEntries(Twinkle.getPref('customWarningList'), sub_group, true);
+			createGroup(Twinkle.warn.messages.singlenotice, 'Single-issue notices', true);
+			createGroup(Twinkle.warn.messages.singlewarn, 'Single-issue warnings', true);
+			createGroup(Twinkle.getPref('customWarningList'), 'Custom warnings', true);
```

Comment:
Not a big deal, but since we're defaulting to `true` these are unnecessary.
```suggestion
			createGroup(Twinkle.warn.messages.singlenotice, 'Single-issue notices');
			createGroup(Twinkle.warn.messages.singlewarn, 'Single-issue warnings');
			createGroup(Twinkle.getPref('customWarningList'), 'Custom warnings');
```